config/i18n.json

```diff
@@ -1348,5 +1351,7 @@
     "WishListNotes": "Wish List Notes: {{notes}}",
     "CopyLine": "Copy Selected Perks as Wish List Roll",
     "CopiedLine": "Wish List roll copied to clipboard"
-  }
+  },
+  "wrong-level": "wrong-level",
+  "no-space": "no-space"
```

Comment:
I don't think these should be included since they're never used as localization keys, see `DimError`'s constructor for this behavior. This caused some confusion in the #i18n channel.